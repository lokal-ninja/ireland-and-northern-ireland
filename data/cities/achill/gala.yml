id: 11104668705
type: node
latitude: 53.97489
longitude: -10.08257
postcode: ""
city: Achill
street: ""
housenumber: 
phone: ""
opening_hours: ""
website: ""