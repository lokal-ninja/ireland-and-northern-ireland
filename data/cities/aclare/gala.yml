id: 1369102481
type: node
latitude: 54.03643
longitude: -8.89886
postcode: ""
city: Aclare
street: ""
housenumber: ""
phone: ""
opening_hours: ""
website: ""
