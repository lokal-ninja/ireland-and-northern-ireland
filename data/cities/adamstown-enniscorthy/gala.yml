id: 264237482
type: way
latitude: 52.39351
longitude: -6.71932
postcode: "Y21 H972"
city: Adamstown, Enniscorthy
street: ""
housenumber: 
phone: "+353 53 9240760"
opening_hours: "Mo-Fr 08:00-21:00; Sa 08:30-21:00; Su 09:00-18:00"
website: "https://www.gala.ie/"