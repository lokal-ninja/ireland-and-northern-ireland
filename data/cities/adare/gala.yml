id: 434902726
type: node
latitude: 52.56201
longitude: -8.79327
postcode: ""
city: Adare
street: Main Street
housenumber: ""
phone: ""
opening_hours: ""
website: ""
