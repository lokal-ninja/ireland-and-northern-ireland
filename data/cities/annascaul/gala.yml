id: 1591172529
type: node
latitude: 52.15241
longitude: -10.05228
postcode: ""
city: Annascaul
street: ""
housenumber: 
phone: ""
opening_hours: ""
website: "https://www.gala.ie/"