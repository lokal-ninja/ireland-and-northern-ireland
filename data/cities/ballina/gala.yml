id: 400894804
type: node
latitude: 54.10595
longitude: -9.16294
postcode: ""
city: Ballina
street: Station Road
housenumber: ""
phone: ""
opening_hours: ""
website: https://www.gala.ie/
