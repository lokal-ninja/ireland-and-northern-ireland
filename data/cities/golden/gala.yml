id: 518605749
type: way
latitude: 52.49877
longitude: -7.98052
postcode: ""
city: Golden
street: ""
housenumber: 
phone: ""
opening_hours: ""
website: ""